pxtrunner/runner.ts

```diff
@@ -213,6 +213,7 @@ namespace pxt.runner {
 
     function initInnerAsync() {
         pxt.setAppTarget((window as any).pxtTargetBundle)
+        pxt.analytics.enable(pxt.Util.userLanguage());
```

Comment:
Not sure about this change? We have other ticks coming in so feels unexpected to have to change this, `enableAnalytics` call should be getting in thru here https://github.com/search?q=repo%3Amicrosoft%2Fpxt%20analytics.enable&type=code

Reply:
If I don't include this line, the `tickEvents` do not make it to Application Insights. See how there are only `share.isMultiplayer` ticks most recently where I commented out this line? Whereas in around 3:04pm I had still had this line enabled and we get the same events mixed in with the new `runner` events.
```suggestion
        pxt.analytics.enable(pxt.Util.userLanguage());
```

![image](https://github.com/microsoft/pxt/assets/6496798/353e5b54-3766-4cf8-8bda-f29174a0bdd5)

Reply:
Adding @thsparks. I have observed that pageview ticks stopped flowing recently as well.

Reply:
@jwunderl There are five files where `pxt.analytics.enable` are called in the linked you provided:

1. Kiosk does not apply in this case since I tested it by opening a share-page after running Arcade locally.
2. I don't believe the share page loads the web app, so telemetry should not be enabled via that route. Though if I'm incorrect, please let me know!
3. Multiplayer page separately enables telemetry, but that does not apply here.
4. Same with the teachertool page.

I can see that argument that if do enable additional telemetry for share pages, then we should match what we've done before in these other examples that you've linked.

Reply:
> Adding @thsparks. I have observed that pageview ticks stopped flowing recently as well.

These changes are unrelated to tracking page views, which are making it. @abchatra